scripts/dts/gen_defines.py

```diff
@@ -154,8 +154,8 @@ def write_props(dev):
             continue
 
         # Skip properties that we handle elsewhere
-        if prop.name in {"reg", "interrupts", "pwms", "clocks", "compatible"} or \
-           prop.name.endswith("gpios"):
+        if prop.name in {"reg", "interrupts", "pwms", "clocks", "compatible", "gpios"} or \
+           prop.name.endswith("-gpios"):
```

Comment:
Looking at it now, I realized this can just be

```python
if prop.name in {"reg", "interrupts", "compatible"}:
    continue
```

, because we never create `Property` objects for properties declared with `type: compound`.

If someone declares e.g. `foo-gpios` as an `int`, then something else is probably broken with their PR, and the check won't do any good in those cases either.

Could you change that while we're here?

Reply:
Done.